src/ch/imedias/rsccfx/view/RsccRequestPresenter.java

```diff
@@ -217,4 +223,28 @@ private void createNewSupporterBtn() {
     }
     attachButtonEvents();
   }
+
+  private void attachContextMenu(Button button) {
+
+    // Create ContextMenu
+    ContextMenu contextMenu = new ContextMenu();
+
+    MenuItem editMenuItem = new MenuItem("Edit");
+    editMenuItem.setOnAction(event -> new SupporterAttributesDialog());
+
+
+    MenuItem callMenuItem = new MenuItem("Call");
```

Comment:
maybe connect would be the better choice here? I would've thought this would mean me to call the person